qpid-jms-client/src/test/java/org/apache/qpid/jms/transports/netty/NettyTcpTransportTest.java

```diff
@@ -394,6 +403,88 @@ public void testZeroSizedSentNoErrors() throws Exception {
         assertTrue(data.isEmpty());
     }
 
+    @Test(timeout = 60 * 1000, expected = IllegalStateException.class)
```

Comment:
Move checking the exception into the test to verify its actually thrown from the intended point.